### DISCUSSION TITLE
NoneBot1 如何获取群成员列表？

### DISCUSSION THREAD

**Discussion by ylca0** [nonebot/discussions#106] [问答]
如题！

**Comment by mnixry** [comment]
获取群成员列表的API是由OneBot端进行实现的, [这里是具体API文档](https://github.com/botuniverse/onebot-11/blob/master/api/public.md#get_group_member_list-%E8%8E%B7%E5%8F%96%E7%BE%A4%E6%88%90%E5%91%98%E5%88%97%E8%A1%A8)
请参考[OneBot 事件和 API](https://docs.nonebot.dev/guide/onebot.html)来进行API调用

---

你可能已经注意到, 我们将您的discussion迁移到了`nonebot/discussion`, 这里是NoneBot问题的综合讨论版, 有任何问题请直接在这里提出
